frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss

```diff
@@ -87,8 +87,11 @@
 }
 
 .co-status-card__health-item-text {
+  word-break: break-word;
   display: block;
   margin-left: 1rem;
+  overflow-wrap: break-word;
+  white-space: pre-line;
```

Comment:
I added these because I noticed that the text wasn't wrapping when the dashboard was full. "Virtualization" was overflowing the container when pseudolocalization was turned on.